mappings/net/minecraft/client/gui/screen/SplashScreen.mapping

```diff
@@ -7,12 +7,20 @@ CLASS net/minecraft/class_425 net/minecraft/client/gui/screen/SplashScreen
 	FIELD field_18219 reloading Z
 	FIELD field_18220 prepareCompleteTime J
 	FIELD field_2483 LOGO Lnet/minecraft/class_2960;
+	FIELD field_25041 bgArgb I
+	FIELD field_25042 bgRgb I
```

Comment:
```suggestion
	FIELD field_25041 BACKGROUND_ARGB I
	FIELD field_25042 BACKGROUND_RGB I
```

Reply:
What's the benefit to making the names that long?

Reply:
These are probably `static final` so captialized. Also `BACKGROUND` is much more explicit than `BG`

Also this is the color of the background?

Reply:
@i509VCB It's the Mojang Studios brand color (the background of the splash screen). Maybe it should just be called `BRAND_[A]RGB` instead?